### PR TITLE
Screenshots and misc small updates

### PR DESCRIPTION
**Pull request by wteouchicago**
- DOC-358: updates url for V2
- DOC-360: removes "coming soon" for Manage Sets
- DOC-334: updates quick start screenshots
- DOC-361: updates name of unsaved cohort
- DOC-363: removes 'coming soon" for body plot
- DOC-365: updates cohort bar and cohort builder image
- DOC-367: updates analysis center image
- DOC-386: updates manage sets in quick start screenshot
- DOC-335: updates cohort builder images
- DOC-336: updates analysis center images, St. Jude tool names
- DOC-395/392/389/390: updates repo and cart images
- DOC-394/393/391: minor text updates to repo and cart, e.g. removes 'coming soon', edited text to match UI changes
- DOC-340: updates projects screenshots
- DOC-364: removes extra paragraph from quick start
- changed "V2" to "2.0" as I encountered them

### CODE REVIEW

File: docs/Data_Portal/Users_Guide/cohort_builder.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Additional features can be accessed at the top right of each card's header to fa
 
 [![Cohort Builder Card Features](images/search_flip_reset_card.png)](images/search_flip_reset_card.png "Image of Cohort Builder Card Features. Click to see the full image.")
 
-In addition, filters in each card can be sorted, either alphabetically or in descending order of number of cases based on current filters, by selecting one of the two icons directly underneath the card title. The default sort is alphabetical order.
+In addition, filters in each card can be sorted, either alphabetically or by the number of cases based on current filters, by selecting one of the two headers directly underneath the card title. The default sort is alphabetical order.
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/cohort_builder.md:34]: i am not loving my phrasing here. please feel free to suggest something better.

File: docs/Data_Portal/Users_Guide/Repository.md [hunk 2]
```diff
@@ -32,13 +32,13 @@ Users may only be interested in browsing through or downloading a subset of file
 * **Platform**: Technological platform on which experimental data was produced.
 * **Access**: Indicator of whether access to the data file is open or controlled.
 
- Values within each facet can be sorted alphabetically by choosing the "AZ" button on the top left of each card. Alternatively, the frequency sort button next to the "Files" labeled may be selected to sort the values by the number of files available.
+ Values within each facet can be sorted alphabetically by choosing the "Name" header on the top left of each card. Alternatively, the the "Files" header may be selected to sort the values by the number of files available.
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/Repository.md:35]: not loving my phrasing here either.

File: docs/Data_Portal/Users_Guide/Repository.md [hunk 3]
```diff
@@ -32,13 +32,13 @@ Users may only be interested in browsing through or downloading a subset of file
 * **Platform**: Technological platform on which experimental data was produced.
 * **Access**: Indicator of whether access to the data file is open or controlled.
 
- Values within each facet can be sorted alphabetically by choosing the "AZ" button on the top left of each card. Alternatively, the frequency sort button next to the "Files" labeled may be selected to sort the values by the number of files available.
+ Values within each facet can be sorted alphabetically by choosing the "Name" header on the top left of each card. Alternatively, the "Files" header may be selected to sort the values by the number of files available.
```
Comment (wteouchicago) [inline docs/Data_Portal/Users_Guide/Repository.md:35]: not loving my phrasing here either

File: docs/Data_Portal/Users_Guide/images/search_flip_reset_card.png
Comment (wwysoc2) [inline docs/Data_Portal/Users_Guide/images/search_flip_reset_card.png:1]: I'll brainstorm ways to not make this shape turn into a full-page feature in the documentation site.

**Review by wwysoc2 — APPROVED** [review]

**Pull request merged**